src/Master.cxx

```diff
@@ -2,11 +2,13 @@
 #include "CommandHandler.h"
 #include "FileWriterTask.h"
 #include "Source.h"
+#include "commandproducer.h"
 #include "logger.h"
 #include <chrono>
-// #include "helper.h"
-// #include "utils.h"
-#include "commandproducer.h"
+#include <rapidjson/document.h>
```

Comment:
What is our policy?
Some projects include the headers which are necessary for the explicitly used types in the source file even if they are already pulled as dependency from another include.
So I use for example explicitly std::chrono, therefore I include <chrono> even though it would be already pulled in through another header.